scripts/test_downgrade_from_tag.sh

```diff
@@ -194,7 +194,7 @@ docker_pgcmd ${CONTAINER_ORIG} "CHECKPOINT;"
 srcdir=$(docker exec ${CONTAINER_ORIG} /bin/bash -c 'pg_config --pkglibdir')
 FILES=$(docker exec ${CONTAINER_ORIG} /bin/bash -c "ls $srcdir/timescaledb*.so")
 for file in $FILES; do
-    docker cp ${CONTAINER_ORIG}:$file ${TEST_TMPDIR}/`basename $file`
+    docker cp ${CONTAINER_ORIG}:$file "${TEST_TMPDIR}/$(basename $file)"
```

Comment:
Do you need quotes around the first parameter too?

Reply:
I can add those as well.